pkg/tsdb/influxdb/influxdb.go

```diff
@@ -106,102 +94,19 @@ func (s *Service) QueryData(ctx context.Context, req *backend.QueryDataRequest)
 	if err != nil {
 		return nil, err
 	}
-	version := dsInfo.Version
-	if version == influxVersionFlux {
-		return flux.Query(ctx, dsInfo, *req)
-	}
-	if version == influxVersionSQL {
-		return fsql.Query(ctx, dsInfo, *req)
-	}
 
-	logger.Debug("Making a non-Flux type query")
+	logger.Debug(fmt.Sprintf("Making a %s type query", dsInfo.Version))
 
-	var allRawQueries string
-	queries := make([]Query, 0, len(req.Queries))
-
-	for _, reqQuery := range req.Queries {
-		query, err := s.queryParser.Parse(reqQuery)
-		if err != nil {
-			return &backend.QueryDataResponse{}, err
-		}
-
-		rawQuery, err := query.Build(req)
-		if err != nil {
-			return &backend.QueryDataResponse{}, err
-		}
-
-		allRawQueries = allRawQueries + rawQuery + ";"
-		query.RefID = reqQuery.RefID
-		query.RawQuery = rawQuery
-		queries = append(queries, *query)
-	}
-
-	if setting.Env == setting.Dev {
-		logger.Debug("Influxdb query", "raw query", allRawQueries)
-	}
-
-	request, err := s.createRequest(ctx, logger, dsInfo, allRawQueries)
-	if err != nil {
-		return &backend.QueryDataResponse{}, err
-	}
-
-	res, err := dsInfo.HTTPClient.Do(request)
-	if err != nil {
-		return &backend.QueryDataResponse{}, err
-	}
-	defer func() {
-		if err := res.Body.Close(); err != nil {
-			logger.Warn("Failed to close response body", "err", err)
-		}
-	}()
-
-	resp := s.responseParser.Parse(res.Body, res.StatusCode, queries)
-
-	return resp, nil
-}
-
-func (s *Service) createRequest(ctx context.Context, logger log.Logger, dsInfo *models.DatasourceInfo, query string) (*http.Request, error) {
-	u, err := url.Parse(dsInfo.URL)
-	if err != nil {
-		return nil, err
-	}
-
-	u.Path = path.Join(u.Path, "query")
-	httpMode := dsInfo.HTTPMode
-
-	var req *http.Request
-	switch httpMode {
-	case "GET":
-		req, err = http.NewRequestWithContext(ctx, http.MethodGet, u.String(), nil)
-		if err != nil {
-			return nil, err
-		}
-	case "POST":
-		bodyValues := url.Values{}
-		bodyValues.Add("q", query)
-		body := bodyValues.Encode()
-		req, err = http.NewRequestWithContext(ctx, http.MethodPost, u.String(), strings.NewReader(body))
-		if err != nil {
-			return nil, err
-		}
+	switch dsInfo.Version {
+	case influxVersionFlux:
+		return flux.Query(ctx, dsInfo, *req)
+	case influxVersionInfluxQL:
+		return influxql.Query(ctx, dsInfo, req)
```

Comment:
The deleted part on the left side moved to the `influxql.go` without changing logic